fundamentals/impl.tex

```diff
@@ -322,13 +322,13 @@ \subsubsection{Resources and Materials}
     \item "never" (currently implemented) globally turns off all decay.
         The Material decay function does nothing.
 
+    \item "lazy" (currently implemented) decays any material whenever its
```

Comment:
got it - done.

On Mon, Feb 23, 2015 at 1:21 PM, Matthew Gidden <notifications@github.com>
wrote:

> In fundamentals/impl.tex
> <https://github.com/cyclus/fundamentals-paper/pull/97#discussion_r25192557>
> :
>
> > @@ -322,13 +322,13 @@ \subsubsection{Resources and Materials}
> >      \item "never" (currently implemented) globally turns off all decay.
> >          The Material decay function does nothing.
> >
> > +    \item "lazy" (currently implemented) decays any material whenever its
>
> e.g., in the previous sentence: "C YCLUS can currently operate in 2 decay
> modes, with 2 additional modes likely to be added in future releases". Can
> you fix that and make sure there aren't others like it, please.
> … <#14bb7e3515259cd1_>
> On Mon, Feb 23, 2015 at 1:16 PM, Robert Carlsen <notifications@github.com>
> wrote: In fundamentals/impl.tex <
> https://github.com/cyclus/fundamentals-paper/pull/97#discussion_r25192135>
> : > @@ -322,13 +322,13 @@ \subsubsection{Resources and Materials} > \item
> "never" (currently implemented) globally turns off all decay. > The
> Material decay function does nothing. > > + \item "lazy" (currently
> implemented) decays any material whenever its I'm not sure what you are
> asking - could you clarify? — Reply to this email directly or view it on
> GitHub <
> https://github.com/cyclus/fundamentals-paper/pull/97/files#r25192135>.
>  -- Matthew Gidden Ph.D. Candidate, Nuclear Engineering The University of
> Wisconsin -- Madison Ph. 225.892.3192
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/fundamentals-paper/pull/97/files#r25192557>.
>